deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```diff
@@ -96,7 +96,7 @@
   value: 3600
 
 {{- range $keyLabel := .Values.ccdb.encryption.rotation.key_labels }}
-{{- if not (regexMatch "^[a-z]+[a-z0-9_]*[a-z0-9]+$" $keyLabel) }}
+{{- if not (regexMatch "^[a-z]+[a-z0-9_]*[a-z0-9]+$" (lower $keyLabel)) }}
 {{ fail "One or more items in ccdb.encryption.rotation.key_labels don't match the regex /^[a-z]+[a-z0-9_]*[a-z0-9]+$/" }}
```

Comment:
Maybe the regexp should be extended to include uppercase letters (instead of using `lower` in the test), so the error message stays correct.

Reply:
Yes, please. Just add the upper-case to the regex.